Bio/Phylo/BaseTree.py

```diff
@@ -152,7 +152,7 @@ def match(node):
             if not hasattr(node, key):
                 return False
             target = getattr(node, key)
-            if isinstance(pattern, basestring):
+            if isinstance(pattern, str):
                 return isinstance(target, basestring) and re.match(
```

Comment:
Missed one, spotted by flake8 and me though.

Reply:
I've changed my environment and missed flake on this one, yeah. I'll fix it in the next commit.